i18n/ru-RU.js

```diff
@@ -125,239 +125,239 @@ module.exports = {
         subscribed_messages: {
-          pre: 'An email to confirm your subscription has been sent to',
+          pre: 'Письмо для подтверждения подписки было отправлено на',
           post: '💚'
         }
       }
     }
   },
   design: {
     meta: {
-      title: 'NuxtJS Design',
+      title: 'NuxtJS Дизайн',
       description:
-        'Download NuxtJS design resources (SVG, icons, emoji and favicon).'
+        'Скачайте дизайнерские файлы NuxtJS (SVG, иконки, emoji и favicon).'
     },
-    title: '{nuxt} Design',
+    title: '{nuxt} Дизайн',
     description:
-      'NuxtJS is an MIT licensed open source project and completely free to use. {break} You can freely use our logos as long as you mention NuxtJS and link to nuxtjs.org.',
+      'NuxtJS - это проект с открытым исходным кодом, лицензированный MIT и полностью бесплатный для использования. {break} Вы можете свободно использоваться наши логотипы пока они ссылаются на nuxtjs.org.',
     other_download_message:
-      'You can also download our {favicon} or our {sketch} file.'
+      'Вы также можете скачать {favicon} или {sketch} файл.'
   },
   resources: {
     meta: {
-      title: 'NuxtJS Resources',
+      title: 'Ресурсы NuxtJS',
       description:
-        'Discover a panel of resources made by our partners. By using thoses affiliate resources links, you are helping us to maintain and develop the Open Source Framework.'
+        'Ознакомьтесь с набором ресурсов сделанных нашими партнерами. Используя эти партнерские ссылки, вы поддерживаете разработку и сопровождение этого фреймворка с открытым исходным кодом.'
     },
     title: '{nuxt} Resources',
     description:
-      'Discover a panel of resources made by our partners. By using thoses affiliate resources links, you are helping us to maintain and develop the Open Source Framework.',
+      'Ознакомьтесь с набором ресурсов сделанных нашими партнерами. Используя эти партнерские ссылки, вы поддерживаете разработку и сопровождение этого фреймворка с открытым исходным кодом',
     themes: {
-      title: 'Themes'
+      title: 'Темы'
     },
     videos: {
-      title: 'Video Courses'
+      title: 'Видео курсы'
     }
   },
   shop: {
     meta: {
-      title: 'The NuxtJS Shop',
+      title: 'Магазин NuxtJS',
       description:
-        'You want to support the NuxtJS project and show your love to the rest of the community? Here our products with the best quality ever!'
+        'Хотите поддержать проект NuxtJS и выразить благодарность его сообществу? Вот наши продукты наилучшего качества!'
     },
-    title: 'The {nuxt} Shop',
+    title: 'Магазин {nuxt}',
     description:
-      'You want to support the NuxtJS project and show your love to the rest of the community?{break} Here our products with the best quality ever!',
-    button: 'Coming soon'
+      'Хотите поддержать проект NuxtJS и выразить благодарность его сообществу?{break} Вот наши продукты наилучшего качества!',
+    button: 'Ожидается'
```

Comment:
```suggestion
    button: 'Скоро'
```
Мне кажется "скоро" тут больше подходит, имхо